test/swarm/run.sh

```diff
@@ -29,6 +29,7 @@ sleep 5
 expect_running_containers "5"
 
 docker stack rm test_stack
+sleep 1
```

Comment:
Does this test fail reliably without this sleep or did you just happen to see it fail intermittently? I think we can keep it, I'd just like to understand it.

Reply:
This was an intermittent fault, probably a timing issue. Didn't run into this before, just added the second of sleeping to hopefully not run into such a timing issue there again.